### ISSUE TITLE
nuxt-auth throwing Error for NextAuthHandler while not imported by project

### ISSUE THREAD

**Issue by otasatan** [sidebase/nuxt-auth#174]
### Environment

------------------------------
- Operating System: `Linux`
- Node Version:     `v16.13.0`
- Nuxt Version:     `3.0.0`
- Nitro Version:    `1.0.0`
- Package Manager:  `npm@8.1.0`
- Builder:          `vite`
- User Config:      `vite`, `modules`, `typescript`, `auth`, `nitro`
- Runtime Modules:  `@sidebase/nuxt-auth@0.3.4`, `@nuxtjs/tailwindcss@6.2.0`
- Build Modules:    `-`
------------------------------

### Reproduction

https://github.com/otasatan/auth

### Describe the bug

The nuxt-auth module throws the Error provided under "Logs".

However, the Project is not importing the NextAuthHandler, as you can see in the provided Repo. Can't start my project due to that error.

### Additional context

I tested it with different Node Versions and also creating the Project with npm create sidebase@latest as well as creating a plain nuxt project and add the module later on.

### Logs

```shell
import { NextAuthHandler } from 'next-auth/core';
         ^^^^^^^^^^^^^^^
SyntaxError: Named export 'NextAuthHandler' not found. The requested module 'next-auth/core' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'next-auth/core';
const { NextAuthHandler } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:181:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async loadESM (node:internal/process/esm_loader:88:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12)
```

**Comment by Sypx** [comment]
Yes, I am also having this issue

**Comment by BracketJohn** [comment]
Will look into it, thanks for the repro 🙇

**Comment by BracketJohn** [comment]
Hey @Sypx and hey @otasatan  🧇 

Thanks for the report - can you see if this problem persists on `@sidebase/nuxt-auth@0.4.0-alpha.3`? Would be nice if you can take this over, should be easy with the repro you provided (:

**Comment by BracketJohn** [comment]
Hey @otasatan 🏴󠁧󠁢󠁷󠁬󠁳󠁿 

Thanks for coming back to this.

> EDIT: Thanks for updating it, it seems to working now except for a stupid error with the parse5 module.

Is this also related to `nuxt-auth`? If so: please open a new issue. I'll close this issue now as resolved by the `0.4.x` update ❤️  @Sypx feel free to also comment if it worked out for you / persists. We can re-open this issue if the latter were the case
